components/brave_vpn/pref_names.cc

```diff
@@ -11,13 +11,18 @@
 namespace brave_vpn {
 namespace prefs {
 
+const char kBraveVPNPurchaseTokenAndroid[] =
```

Comment:
if we are not registering `kBraveVPNPurchaseTokenAndroid` outside of android then we can never safely access it outside of android so this should also be guarded

Reply:
also you're missing build_config.h where `BUILDFLAG(IS_ANDROID)` is defined

Reply:
Fixed; added build_config.h include and then updated GetPurchaseToken method to only exist on Android

Reply:
this should also have a guard so it can't accidentally be accessed

Reply:
fixed